Hyperion, a building event map

png103 opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## What Does This PR Do
<!-- Include a small to medium description of what your PR changes. Document all changes, as not doing this may delay reviews or even discourage maintainers from merging your PR! -->
I've been working on for a week at this point, many thanks to Syrrian for giving feedback throught and S34N for doing the piping/wiring

After nanotrasen's engineering team failed to meet time frames on the construction of their next station, Instead of canceling the shift and giving everyone the day off they chose to disguise the unfinished station as a "fun" company event where they'd just let whoever the hell wanted to finish the construction for them, isn't extra unpaid labor sweet?

The map only contains the bare minimum for everything to not spiral instantly into complete anarchy,

- A kitchen, so the crew doesn't starve to death
- Botany (or Hydroponics if you prefer so)
- Janitor's closet to clean up the inevitable messes
- Cargo to order the usual silly things
- Engineering supply closet to generate power
- Brig to give security something to do
- RnD to make electronics
- A table and two chairs for bridge hobos

There's a list of things that the crew is assigned to build, included in a note that an admin would spawn in bridge at round start

`[center][logo][/center][br]Dear capt.,[br]We regret to inform you that our team of skilled engineers failed to complete the station's construction in time, And as a result, you may have noticed that it is somewhat... incomplete.[br][br]So. after a few days of deliberation and planning, we are delighted to announce that Hyperion station has been cancelled, and on it's place, we present: project Build Your Own Nanotrasen Depot (B.Y.O.N.D.), Your goal is to encourage the crew to assist in the expansion and construction of the station; we have provided you with the necessary materials and tools, higger ups will be sent later to review your progress and give the final score. If you require anything specific, please contact central command, but keep in mind that we will only accept requests once.[br][br]The station necessitates the following:[br][br]-Rooms with proper wiring/lighting/air flow[br]-Clean and tidy departments[br]-A place for dining/drinking (preferably with actual food and drinks)[br]-A medical center[br]]-A waste disposal system[br]-A conference room[br][br][br]Extra points.[br][br]The features listed bellow are optional, however they will help improve your rating.[br][br]-A theatre or stage[br]-Dormitories[br]-A locker room[br]-A chapel or church[br]-A library[br]-A recreational room[br][br]So, without further ado, we wish you all the best of luck.[br][br][i]-Do not let us down!, Glory to Nanotrasen Inc![/i]`

I've ran the map several times, from what i've seen everything works as intended, but if any changes are needed i'll (try to) comply
## Why It's Good For The Game
<!-- Add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->
Encouraging players to build stuff is fun
## Images of changes
<!-- If you did not make a map or sprite edit, you may delete this section. You may include a gif or mp4 of your feature if you want. -->
Arrival shuttle (mining shuttle docking on the left and mini medbay on the right)
![image](https://user-images.githubusercontent.com/68801329/147798873-9bc42dae-b0ac-4b90-9600-a63329335b0d.png)
Station
![image](https://user-images.githubusercontent.com/68801329/147798863-462ad60c-b688-4d66-ac95-2d777be53af6.png)
Close ups
![image](https://user-images.githubusercontent.com/68801329/147798946-4d655d03-d063-4fee-8a00-240a82a5329d.png)
![image](https://user-images.githubusercontent.com/68801329/147798961-77c3d804-7dd4-44e0-9ce8-e7637e105a22.png)
![image](https://user-images.githubusercontent.com/68801329/147798969-21cbf8e5-6dc5-498a-822f-1fd908e14947.png)
![image](https://user-images.githubusercontent.com/68801329/147798975-6e6e7bbf-6183-463e-8679-5755560e14a8.png)
![image](https://user-images.githubusercontent.com/68801329/147799130-cd7978c7-e76b-41f4-8275-f1923f590bbb.png)



🆑 S34NW, Sirryan2002, png103
Add: new map
/ 🆑

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->
<!-- If a PR has no impact on players (i.e. a code refactor that does not change functionality) then the entire Changelog heading and contents can be removed. -->

Sirryan2002 commented:
Just to be clear to those unaware, if given the go ahead by maints/host, this will be TM'd once and then closed afterwards. This is not going to be a permanent feature addition.

Denthamos commented:
I like the idea, though considering how high traffic the arrivals shuttle will be already due to it being a spawn in location and the only functional on site medical area at the start, if the mining area is in a bad spot.  Having where miners will be constantly going back and forth through, where the station personnel will constantly be going to in order to get metal and glass, and such will make that area a bit too congested for such uses.  

At the same time however not sure where else would be a good spot for it to dock at.

png103 commented:
> I like the idea, though considering how high traffic the arrivals shuttle will be already due to it being a spawn in location and the only functional on site medical area at the start, if the mining area is in a bad spot. Having where miners will be constantly going back and forth through, where the station personnel will constantly be going to in order to get metal and glass, and such will make that area a bit too congested for such uses.
> 
> At the same time however not sure where else would be a good spot for it to dock at.

i've placed down plenty of metal in the station, you can see it in the tables on the first close up, so the orm would probably only be used for emergencies when they are REALLY out, or just for plasma and all that so i bet it wouldn't be that crowded

png103 commented:
if a host/maintainer is reading this, i'm still working on improving the map so the one currently on the branch is not that final but it's pretty close

S34NW commented:
Only minor mapping issues present, will be fine for an event shift.

hal9000PR commented:
For changelogs you can put the names of all that worked on it next to the first :cl:
Eg. 
🆑 S34N, sirryan, png103
Add: new map
/ 🆑

png103 commented:
alright so, i fixed some mistakes with the map, added an atmos storage and made this better, also added an art vendor so crew can get crayons and all
![image](https://user-images.githubusercontent.com/68801329/147978622-0108b44d-50cd-498a-be8d-4907a5b3fed8.png)
![image](https://user-images.githubusercontent.com/68801329/147978508-676def15-597b-4c20-9def-97a1d49e13bc.png)
![image](https://user-images.githubusercontent.com/68801329/147978580-23f58628-afbd-4b36-a709-74d34b1f4231.png)

Denthamos commented:
It would be neat to see how this works out otherwise for a planned event.  Throwing my approval on it as long as you properly coordinate a planned event time for this to happen and arrange for it to be announced on discord for players to know when the test merge and event is happening.

dearmochi reviewed (changes requested):
Please revert the Cyberiad changes -- the map `hyperion` datum will be used for the admin to force switch to it when it's event time.

dearmochi commented on code/modules/mapping/cyberiad.dm:
```diff
@@ -1,5 +1,5 @@
 /datum/map/cyberiad
-	fluff_name = "NSS Cyberiad"
-	technical_name = "Cyberiad"
+	fluff_name = "Hyperion"
+	technical_name = "Project B.Y.O.N.D"
```
```suggestion
	fluff_name = "NSS Cyberiad"
	technical_name = "Cyberiad"
```

This still needs reverting.

png103 commented:
oops forgot about that

png103 commented:
borg recharging pod and like 5 rcds to avoid atleast a little bit of metal to be used up on the autolathes (also forgot to include some pictures of the cryo pods
![image](https://user-images.githubusercontent.com/68801329/149623130-394431c8-eedc-4ffa-ad56-d2da1ce86d05.png)
![image](https://user-images.githubusercontent.com/68801329/149623133-d018e0c8-31e9-4642-ba82-e9db717a1e2f.png)
![image](https://user-images.githubusercontent.com/68801329/149623203-ad667e28-754f-4b9f-ab9b-9ef3d2927830.png)

AffectedArc07 commented on code/modules/mapping/hyperion.dm:
```diff
@@ -0,0 +1,4 @@
+/datum/map/hyperion
+	fluff_name = "Hyperion"
+	technical_name = "Project B.Y.O.N.D"
```
```suggestion
	technical_name = "hyperion"
```

AffectedArc07 commented:
![image](https://user-images.githubusercontent.com/25063394/149626399-6b42cce5-3f00-46f4-975d-c22a437ab475.png)

png103 commented:
i made some changes to fix the z/w issues and also fixed some spawn issues i'm testing it rn before i add it to the branch

png103 commented:
i still have a changes requested warning despite doing the requested change
![image](https://user-images.githubusercontent.com/68801329/149633239-bbc0c4f6-23fa-4152-8484-a8a617910d90.png)

png103 commented:
by the way this isn't coding related but i need 2/3 admins that would play as the inspectors at the end of the round
also if you're gonna be on the event dm me (Peterr#9201) so i can send you a google docs with some basic midround events and timings

png103 commented:
that was really fun, i'm glad it went (mostly) well
